source/MaterialXGenShader/ColorManagementSystem.h

```diff
@@ -72,6 +78,23 @@ class MX_GENSHADER_API ColorManagementSystem
     /// Returns an implementation for a given transform
     virtual ImplementationPtr getImplementation(const ColorSpaceTransform& transform) const = 0;
 
+    /// For a given transform node and a target port type to connect to, determine the appropriate
+    /// input and output ports. If the transform node's port type does not match the target type
+    /// additional "conversion" nodes will created and the appropriate input and output ports on these 
+    /// nodes will be returned.
+    ///
+    /// Possible configirations include the first two cases where no additional conversion nodes are required, and
```

Comment:
Typo: configirations

Reply:
Also, should be: nodes will **be** created

Reply:
Will fix.